web/server/entrypoint.go

```diff
@@ -333,8 +333,9 @@ func (s *robotServer) configWatcher(ctx context.Context, currCfg *config.Config,
 ) {
 	// Reconfigure robot to have passed-in config before listening for any config
 	// changes.
+	startTime := time.Now()
 	r.Reconfigure(ctx, currCfg)
-
+	s.logger.CInfow(ctx, "Robot initialized with full config", "elapsed time", time.Since(startTime).String())
```

Comment:
These "elapsed time" values here and for `"Robot created with minimal config"` are a little strange in terms of wording. It's not really the time "elapsed" _since_ anything. It's more like the amount of time it took to call `New` and `Reconfigure`? Maybe we could use the field names "time to create" and "time to reconfigure" respectively?

Reply:
updated